master/buildbot/status/build.py

```diff
@@ -557,6 +555,15 @@ def upgradeToVersion4(self):
             del self.source
         self.wasUpgraded = True
 
+    def upgradeToVersion5(self):
+        if not hasattr(self, "user_id"):
+            self.user_id = None
+            # maybe we should search here user_id 'in old way'
```

Comment:
I think we should search user_id in old way in upgrade method 👍

Reply:
Yup, I've uncommented this code.